# 江苏 js 浙江 zj 安徽 ah 河南 he 湖南 hu 四川 sc 广东 gd 福建 fj 无效 wx
# 江苏 1  浙江 2  安徽 3  河南 4  湖南 5  四川 6  广东 7  福建 8  无效 0
int32 type
#左 1/1.1 1/1.2 2/1.3 2/1.4 3/1.5 中 4/1.6 5/1.7 5/1.8 6/1.9 6/1.10 右
#左 1/2.1 1/2.2 2/2.3 2/2.4 3/2.5 中 4/2.6 5/2.7 5/2.8 6/2.9 6/2.10 右
---
int32[] results
int32[] positions_z #1 2
int32[] positions_x #1 2 3 4 5 6 7 8 9 10